src/attributes/limits.md

```diff
@@ -7,8 +7,42 @@ The following [attributes] affect compile-time limits.
 The *`recursion_limit` attribute* may be applied at the crate level to set the
 maximum depth for potentially infinitely-recursive compile-time operations
 like auto-dereference or macro expansion. It uses the [_MetaNameValueStr_]
-syntax to specify the recursion depth. The default is
-`#![recursion_limit="64"]`.
+syntax to specify the recursion depth. The default is 64.
```

Comment:
That default is rustc specific, and either be in a note or we should link to a corresponding page on the rustc book.